### DISCUSSION TITLE
Action "packs.install" cannot be found. for url: http://sts-st2api:9101/packs/install

### DISCUSSION THREAD

**Discussion by sharhan-alhassan** [StackStorm/st2#5880] [Operations]
I am currently having issues installing any pack at all in stackstorm using the st2 client. 

Error:

```sh
ERROR: 400 Client Error: Bad Request
MESSAGE: Action "packs.install" cannot be found. for url: http://sts-st2api:9101/packs/install
```

Environment: Running Kubernetes on local kind cluster

To repoduce: 

Exec into the `st2 client` pod

Then login to the st2 using: `st2 login -w st2admin`

Install a pack: `st2 pack install https://github.com/emedvedev/chatops_tutorial`

**Comment by arm4b** [comment]
Check the https://github.com/StackStorm/stackstorm-k8s/#install-custom-st2-packs-in-the-cluster
as K8s HA architecture for making the pack install work requires more configuration from your side.

**Reply by sharhan-alhassan** [reply]
Hey @armab 
Great feedback. I went with the second approach with volumes. However, installing a pack just hangs in there and takes forever to complete.

```sh
root@sts-st2client-6b7d76f4b8-tzrwc:/opt/stackstorm# st2 pack install git

For the "git" pack, the following content will be registered:

actions   |  7
rules     |  1
sensors   |  1
aliases   |  0
triggers  |  0

Installation may take a while for packs with many items.

```

**Comment by arm4b** [comment]
Check the st2api and st2actionrunner logs to understand where it's stuck.

Additionally, what's your K8s configuration, some cloud? How the NFS was configured with the volumes?

**Reply by sharhan-alhassan** [reply]
I'm running Kubernetes from local kind clusters

The st2runner error is shown below:
```sh
amqp.exceptions.NotFound: Queue.declare: (404) NOT_FOUND - home node 'rabbit@sts-rabbitmq-0.sts-rabbitmq-headless.default.svc.cluster.local' of durable queue 'st2.preinit' in vhost '/' is down or inaccessible
2023-02-04 09:57:34,983 ERROR [-] Failed to predeclare queue for exchange "st2.sensor"
```
